src/univariate_milp_relaxation.jl

```diff
@@ -47,18 +71,23 @@ function _build_univariate_milp_relaxation!(
         sec_vs[1][2] + sum(
             delta_1[i] * (tan_vs[i][2] - sec_vs[i][2]) +
             delta_2[i] * (sec_vs[i+1][2] - sec_vs[i][2]) for i = 1:num_vars
-        )
+        ),
+        base_name = constraint_pre_base_name * "y"
     )
 
     # add first delta constraint
-    formulation_info.constraints[:first_delta] =
-        @constraint(m, delta_1[1] + delta_2[1] <= 1)
+    formulation_info.constraints[:first_delta] = 
+        @constraint(m, delta_1[1] + delta_2[1] <= 1,
+            base_name = constraint_pre_base_name * "first_delta")
 
     # add linking constraints between delta_1, delta_2 and z
-    formulation_info.constraints[:below_z] =
-        @constraint(m, [i = 2:num_vars], delta_1[i] + delta_2[i] <= z[i-1])
-    formulation_info.constraints[:above_z] =
-        @constraint(m, [i = 2:num_vars], z[i-1] <= delta_2[i-1])
+    formulation_info.constraints[:below_z] = 
+        @constraint(m, [i = 2:num_vars], delta_1[i] + delta_2[i] <= z[i-1],
+            base_name = constraint_pre_base_name * "below_z")
+
+    formulation_info.constraints[:above_z] = 
+        @constraint(m, [i = 2:num_vars], z[i-1] <= delta_2[i-1],
+            base_name = constraint_pre_base_name * "above_z")
 
     return formulation_info
 end
```

Comment:
Are you planning to provide on/off functionality for MILP relaxations in a future release?

Reply:
I have not figured out the right formulation. The first step was LP relaxations which is easy. After we figure it out, yes, we will add it.